azure-pipelines/steps/install-development-dependencies.yml

```diff
@@ -12,5 +12,6 @@ steps:
       python -m pipenv lock --dev -r --pre > dev-requirements.txt
       pip install -r dev-requirements.txt
       pip install pytest-azurepipelines
+      pip install towncrier
```

Comment:
Should towncrier be in `dev-requirements.txt` instead?

Reply:
yeah moved `towncrier` required to `Pipfile`.